test/e2e/explainer/test_aix_explainer.py

```diff
@@ -0,0 +1,88 @@
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import os
+
+from kubernetes import client
+
+from kfserving import KFServingClient
+from kfserving import constants
+from kfserving import V1alpha2EndpointSpec
+from kfserving import V1alpha2PredictorSpec
+from kfserving import V1alpha2InferenceServiceSpec
+from kfserving import V1alpha2ExplainerSpec
+from kfserving import V1alpha2AIXExplainerSpec
+from kfserving import V1alpha2InferenceService
+from kfserving import V1alpha2CustomSpec
+from kubernetes.client import V1ResourceRequirements
+from kubernetes.client import V1Container
+
+from ..common.utils import predict
+from ..common.utils import explain_aix
+from ..common.utils import KFSERVING_TEST_NAMESPACE
+
+import numpy as np
+
+logging.basicConfig(level=logging.INFO)
+api_version = constants.KFSERVING_GROUP + '/' + constants.KFSERVING_VERSION
+KFServing = KFServingClient(config_file=os.environ.get("KUBECONFIG", "~/.kube/config"))
+
+
+def test_tabular_explainer():
+    service_name = 'aix-explainer'
+    default_endpoint_spec = V1alpha2EndpointSpec(
+        predictor=V1alpha2PredictorSpec(
+            custom=V1alpha2CustomSpec(
+                container=V1Container(
+                    name="predictor",
+                    image='aipipeline/rf-predictor:0.4.0',
+                    command=["python", "-m", "rfserver", "--model_name", "aix-explainer"],
+                    resources=V1ResourceRequirements(
+                        requests={'cpu': '1', 'memory': '2Gi'},
+                        limits={'cpu': '1', 'memory': '2Gi'})
+                ))),
+        explainer=V1alpha2ExplainerSpec(
+            min_replicas=1,
+            aix=V1alpha2AIXExplainerSpec(
+                type='LimeImages',
+                resources=V1ResourceRequirements(
+                    requests={'cpu': '1', 'memory': '2Gi'},
+                    limits={'cpu': '1', 'memory': '2Gi'}))))
+
+    isvc = V1alpha2InferenceService(api_version=api_version,
+                                    kind=constants.KFSERVING_KIND,
+                                    metadata=client.V1ObjectMeta(
+                                        name=service_name, namespace=KFSERVING_TEST_NAMESPACE),
+                                    spec=V1alpha2InferenceServiceSpec(default=default_endpoint_spec))
+
+    KFServing.create(isvc)
+    try:
+        KFServing.wait_isvc_ready(service_name, namespace=KFSERVING_TEST_NAMESPACE, timeout_seconds=720)
+    except RuntimeError as e:
+        logging.info(KFServing.api_instance.get_namespaced_custom_object("serving.knative.dev", "v1",
+                     KFSERVING_TEST_NAMESPACE, "services", service_name + "-predictor"))
```

Comment:
oh the service name is actually, `service_name + "-predictor-default"`, I was trying to remove `-default` but figured it breaks the backwards compatibility.

Reply:
Thanks, we added back -default